exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerBuilder.java

```diff
@@ -83,10 +83,11 @@ public PrometheusHttpServerBuilder setPrometheusRegistry(PrometheusRegistry prom
     return this;
   }
 
-  /** Set if the {@code otel_scope_*} attributes are generated. Default is {@code true}. */
+  /** Set if the {@code otel_scope_info} metric is emitted. Default is {@code true}. */
   @SuppressWarnings("UnusedReturnValue")
-  public PrometheusHttpServerBuilder setOtelScopeEnabled(boolean otelScopeEnabled) {
-    this.otelScopeEnabled = otelScopeEnabled;
+  public PrometheusHttpServerBuilder setOtelScopeInfoMetricEnabled(
```

Comment:
probably should do a deprecation cycle for the previous public method?

otoh, I don't mind the old method name, seems reasonably descriptive, wdyt?